app/controllers/prizes_controller.rb

```diff
@@ -0,0 +1,20 @@
+class PrizesController < ApplicationController
+  def index
+    render json: Prize.all
+  end
+
+  def create
+    prize = Prize.create(prize_params)
+    render json: prize
+  end
+
+  def win_prize
+    render json: Prize.where(image: 'Logo  2')
```

Comment:
`Logo  2` seems to have to spaces. Also, can we please be more descriptive in naming? What's logo 2?

Reply:
That's a test I forgot to remove, I'll make sure to remove that tomorrow. That should be some logic to give a prize at random according to the probabilities.